mne_realtime/base_client.py

```diff
@@ -43,19 +43,23 @@ class _BaseClient(object):
         Time instant to stop receiving buffers.
     buffer_size : int
         Size of each buffer in terms of number of samples.
+    host_name : str | None
+        The name of the server, e.g. the LSL stream name.
+        If None, the name is ignored.
     %(verbose)s
     """
 
     def __init__(self, info=None, host='localhost', port=None,
                  wait_max=10., tmin=None, tmax=np.inf,
-                 buffer_size=1000, verbose=None):  # noqa: D102
+                 buffer_size=1000, host_name=None, verbose=None):  # noqa: D102
```

Comment:
In MNE-Python we started adding `*` to stop people from using long `arg` lists and getting errors, seems useful here, too
```suggestion
                 buffer_size=1000, *, host_name=None,
                 verbose=None):  # noqa: D102
```